File: public/snippets/C/Looping Do...While.c
#include <stdio.h>

/*	------------------------------------------	*/
/*	 SYNTAX LOOPING DO...WHILE PADA BAHASA C 	*/
/*	------------------------------------------	*/
/*			do {								*/			
/* 	  			statement(s);					*/
/*			} while( condition );				*/
/*	------------------------------------------	*/
 
int main () {

   /* Deklarasi variabel lokal */
   int a = 10;

   /* Eksekusi Do..While Looping dilakukan */
   do {
      printf("value of a: %d\n", a);
      a = a + 1;
   }while( a < 20 );
 
   return 0;
}

File: public/snippets/C#/Looping Do...While.cs
using System;

/* ------------------------------------------- */
/*  SYNTAX LOOPING DO...WHILE PADA BAHASA C#   */
/* ------------------------------------------- */
/*       do {                                  */
/*             statement(s);                   */
/*       } while( condition );                 */
/* ------------------------------------------- */

namespace Loops
{
   class Program
   {
      static void Main(string[] args)
      {
         /* Deklarasi Variabel Lokal */
         int a = 10;
         
         /* Do..While dilakukan */
         do
         {
            Console.WriteLine("value of a: {0}", a);
            a = a + 1;
         } 
         while (a < 20);
         Console.ReadLine();
      }
   }
} 